plugins/lookup/conjur_variable.py

```diff
@@ -203,12 +210,24 @@ def _fetch_conjur_variable(conjur_variable, token, conjur_url, account, validate
     return {}
 
 
+def _store_in_file(value):
+    file_name = ''.join(choice(string.ascii_uppercase + string.digits) for i in range(12))
+    file_path = os.path.join("/dev/shm", file_name)
+    with open(file_path, 'w+') as file:
```

Comment:
`+` in the mode isn't needed since we won't be reading the file

Reply:
`file` -> `secrets_file`

Reply:
I agree!